cluster/src/main/scala/io/snappydata/gemxd/SparkSQLPrepareImpl.scala

```diff
@@ -225,6 +241,22 @@ class SparkSQLPrepareImpl(val sql: String,
   }
 }
 
+object SparkSQLPrepareImpl{
+
+  def getTableNamesAndDatatype(output: Seq[expressions.Attribute]):
+  (Array[String], Array[DataType]) = {
+    var i = 0
```

Comment:
Can be a simple map function which can remove the var i;

Reply:
Yes. Let me change it. I just focused on correctness in Vivek's changes.